README.md

```diff
@@ -109,7 +109,13 @@ log-20160701.txt
 log-20160702.txt
 ```
 
-To prevent outages due to disk space exhaustion, each file is capped to 1 GB in size. If the file size is exceeded, events will be dropped until the next roll point.
+To prevent outages due to disk space exhaustion, each file is capped to 1 GB in size. If the file size is exceeded, events will be dropped until the next roll point or new file with number appended in the format <code>_NNN</code>, with the first filename given no number, if rollOnFileSizeLimit is true.
```

Comment:
Since it's a non-default option, I think we can keep this simple and add the new information under the documentation for the `rollOnFileSizeLimit` parameter only.

Reply:
removed